### DISCUSSION TITLE
Do NOT upgrade to Ubuntu 23.04 at this time

### DISCUSSION THREAD

**Discussion by squassina** [Joshua-Riek/ubuntu-rockchip#515] [General]
Hi, I'm glad for this distro, and I'm reinstalling right now!

I saw a prompt that there was an update available, so I happily and naively updated it, just to see a lot of errors, really a lot, and the device won't reboot. I had to press the reset button, so I can see a Kernel Panic in the end of the failed boot.

So, learn from the errors of others and don't update at this time.

**Comment by Joshua-Riek** [comment]
Ah, sorry about that, I made a system change to disable this update message. I will take a look at other options to further disable an update prompt.

RK3588 will likely be stuck on Ubuntu 22.04 for a while because of panfork. Upgrading, at least for now, will break hardware acceleration.

**Reply by squassina** [reply]
Thanks!
